bfxapi/models/funding_ticker.py

```diff
@@ -0,0 +1,93 @@
+"""
+Module used to describe all of the different data types
+"""
+
+class FundingTickerModel:
+    """
+    Enum used to index the different values in a raw funding ticker array
+    """
+    FRR = 0
+    BID = 1
+    BID_PERIOD = 2
```

Comment:
Bid/ask period/size are reversed relative to `bfx-api-node-models`: https://github.com/bitfinexcom/bfx-api-node-models/blob/master/lib/funding_ticker.js#L13

Reply:
Hmmmm maybe the docs are incorrect then, I copied the indexes from https://docs.bitfinex.com/reference#ws-public-ticker

ill test

Reply:
Response from websocket:

```
[0.00022647945205479453, 0.0001802, 30, 1561871.6180378501, 0.0001656, 2, 310554.22850161, -2.197e-05, -0.089, 0.0002248, 108955335.67687598, 0.00035, 1e-06, None, None, 4661091.26128378]
```

It seems like bid period is at index `3` and size is at `4`

Reply:
Alright, will fix in the node lib :+1: Good find